src/rust/Dockerfile

```diff
@@ -92,7 +92,7 @@ RUN cp -ar /tmp/target/. /grapl/rust/target/ && \
     cp -ar /tmp/registry/. /usr/local/cargo/registry/ && \
     rm -rf /tmp/{target,registry}
 
-CMD [ "bash", "-c", "cargo test --features \"${RUST_INTEGRATION_TEST_FEATURES}\" --test \"*\"" ]
+CMD [ "bash", "-c", "cargo test --features \"${RUST_INTEGRATION_TEST_FEATURES}\" --test \"*\" -- --nocapture" ]
```

Comment:
Torn on whether I want to disable this or not

Reply:
I think its worth keeping this. It makes logging easier and then its a choice for the dev on whether they want to add logs or not